README.md

```diff
@@ -1,24 +1,234 @@
-# README
+<a name="readme-top"></a>
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
 
-Things you may want to cover:
+Modify this file to match your project and remove sections that don't apply.
 
-* Ruby version
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
 
-* System dependencies
+After you're finished please remove all the comments and instructions!
+-->
 
-* Configuration
+<div align="center">
 
-* Database creation
+  <img src="https://user-images.githubusercontent.com/84629565/202665566-ba1a8ed3-041f-45bc-b21b-efdcc357189b.png" alt="logo" width="140"  height="auto" />
+  <br/>
 
-* Database initialization
+  <h3><b>Microverse README Template</b></h3>
 
-* How to run the test suite
+</div>
 
-* Services (job queues, cache servers, search engines, etc.)
+<!-- TABLE OF CONTENTS -->
 
-* Deployment instructions
+# 📗 Table of Contents
 
-* ...
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Budget-Place <a name="about-project"></a>
+The Budget place 
+An app to manager your own budget.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Ruby</summary>
+</details>
+
+<details>
+  <summary>Ruby on Rails</summary>
+</details>
+
+<details>
+  <summary>Tailwind CSS </summary>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[Authentication, User can manage his recepies and foods]**
+- **[Guest User can navigate the App and read all ]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Budget Place](https://budget-place.onrender.com/)
```

Comment:
- [ ] As it was requested of you by the [previous reviewer](https://github.com/Amtresu/Budget-Place/pull/1/files#r1052919988) and as per the project requirements, kindly ensure that you have deployed your application so that it can be accessible by everyone online.

![image](https://user-images.githubusercontent.com/86318284/208605357-f5780897-2be1-41d9-9e75-4143115e0c08.png)


**This is what this link shows:**

![image](https://user-images.githubusercontent.com/86318284/208607706-41f2516c-0bab-4f20-bfb8-55eb42fb86d4.png)